pkg/kv/store_message_test.go

```diff
@@ -19,8 +19,7 @@ const modelPrefix = "tm"
 
```

Comment:
I think we're missing tests for the new Iterators and all the possible states that the store may be at in any given time.

Reply:
Added